source/examples/rapids-sagemaker-higgs/.dockerignore

```diff
@@ -0,0 +1,6 @@
+# ignore everything by default
+*
+
+# except these specific things
+!entrypoint.sh
+!rapids-higgs.py
```

Comment:
The `rapids-sagemaker-higgs` notebook runs `docker build` in a notebook cell. Without the introduction of this `.dockerignore`, it ends up sending around 10GB (!) of context to `docker build`, which adds about 30 seconds to the build.